Makefile

```
@@ -19,6 +19,11 @@ YAMLLINT_CONFIG ?= .yamllint.yml
 YAMLLINT_IMAGE  ?= docker.io/cytopia/yamllint:latest
 YAMLLINT_DOCKER ?= $(DOCKER_CMD) $(DOCKER_ARGS) $(YAMLLINT_IMAGE)
 
+COMPONENT_NAME ?= $(shell basename ${PWD} | sed s/component-//)
+COMMODORE_CMD  ?= docker run --rm --user="$(shell id -u)" --volume "${PWD}:/app/$(COMPONENT_NAME)" --workdir /app/$(COMPONENT_NAME) projectsyn/commodore:latest component compile . -J vendor -f tests/test.yml
```

I think the `vendor` dir is used by default.

```suggestion
COMMODORE_CMD  ?= docker run --rm --user="$(shell id -u)" --volume "${PWD}:/app/$(COMPONENT_NAME)" --workdir /app/$(COMPONENT_NAME) projectsyn/commodore:latest component compile . -f tests/test.yml
```